src/python/WMCore/Services/Rucio/Rucio.py

```diff
@@ -789,6 +789,22 @@ def pickRSE(self, rseExpression='rse_type=TAPE\cms_type=test', rseAttribute='ddm
         choice = weightedChoice(rsesWithWeights)
         return choice
 
+    def requiresApproval(self, rse):
+        """
+        _requiresApproval_
+
+        Returns wether or not the specified RSE requires rule approval.
+        :param res: string containing the Rucio RSE name
+        :returns: True if the RSE requires approval to write (rule property)
+        """
+        try:
+            attrs = self.cli.list_rse_attributes(rse)
+        except RSENotFound as exc:
+            msg = "Error retrieving attributs for RSE: {}. Error: {}".format(rse, str(exc))
```

Comment:
Can you please fix this typo "attributs"?

Reply:
Done!

Reply:
Thanks!